debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/SnapshotReader.java

```diff
@@ -194,6 +202,12 @@ protected void execute() {
                         // This column exists only in MySQL 5.6.5 or later ...
                         String gtidSet = rs.getString(5);// GTID set, may be null, blank, or contain a GTID set
                         source.setGtidSet(gtidSet);
+                        if (logger.isDebugEnabled()) {
+                            logger.debug("Step 3: using binlog '{}' at position '{}' and gtid '{}'", binlogFilename, binlogPosition,
+                                         gtidSet);
+                        }
+                    } else if (logger.isDebugEnabled()) {
+                        logger.debug("Step 3: using binlog '{}' at position '{}'", binlogFilename, binlogPosition);
```

Comment:
The "Step 3" prefix is intended to appear just once; simply indent for log messages within that same section.